src/content/overview/observability/observability-platform-api/_index.md

```diff
@@ -0,0 +1,105 @@
+
+The `observability platform API` is an additional component that is a part of the platform, and it allows customers to ingest their logs (for now) from outside their clusters into our observability platform. Moreover, it also allows them to access our own observability data through their own Grafana running on their premises.
+
+Its main objectives are to be able to:
+
+* provide a secure access to our observability platform and reduce the attack surface.
+* allow customers to access our observability data and to send their own.
+* sanitize (sampling, relabelling, ...) observability data before it reaches our backends.
+* support OTLP (OpenTelemetry).
+
+## Architecture
+
+### Global overview
+
+Here is a graph showing the general architecture for the API :
+
+![api architecture](./observability-platform-api-graph.png)
+
+### How it works
+
+Let's explore the 2 main workflows from the customer's perspective.
+
+#### Read path
+
+Let's consider the following situation : a customer wants to access Giant Swarm observability data from his own Grafana running on his premises.
+
+To do so, the customer will first log into the Grafana instance using the OIDC provider hosted on his premises. Once authenticated, he will perform a query from the "Explore" panel or simply use dashboards which in any case will trigger a request sent to the Observability Platform API url.
+
+There, depending on the path the requests is trying to access (i.e either trying to access logs or metrics) it will go through one of 2 ingresses sharing the same host : one in the loki namespace and the other in the mimir one. Note that we decided to have 2 ingresses sharing the same host instead of having 1 ingress handling all paths because usually, ingresses only forward requests in the same namespace they are deployed in. There are ways to go around this, but those are a bit tricky and definitely not straight forward.
+
+Upon reaching the correct ingress, the request will need to be authenticated before being forwarded. This is done through a nginx annotation : `nginx.ingress.kubernetes.io/auth-url` which value is the customer OIDC provider ingress' url. As the requests coming from Grafana forwards the user's OAuth identity through the use of a token, the Giant Swarm ingress only make sure that the incoming request's token matches an allowed user in the OIDC provider.
+
+Moreover, the ingress will verify that the incoming requests are using the `X-Scope-OrgId` HTTP headers (tenant information). If not, the request will be denied.
+
+Upon authentication, the request is then forwarded to either the `loki-gateway` or `mimir-gateway` (depending if the query is looking for logs or metrics) components which are nginx servers respectively fronting the loki and mimir clusters. Those gateway components then finally forward the request to the read components.
```

Comment:
Again...very technical details customers can also just read from the diagram if they are interested in it (or ask us). This doesn't add too much value if you just want to read data from somewhere.